docs/diagrams/UiClassDiagram.puml

```diff
@@ -32,28 +35,36 @@ UiManager .left.|> Ui
 UiManager -down-> MainWindow
 MainWindow --> HelpWindow
 MainWindow *-down-> CommandBox
+MainWindow *-down-> StatisticsDisplay
 MainWindow *-down-> ResultDisplay
 MainWindow *-down-> PersonListPanel
+MainWindow *-down-> AppointmentListPanel
 MainWindow *-down-> StatusBarFooter
 
 PersonListPanel -down-> PersonCard
+AppointmentListPanel -down-> AppointmentCard
 
 MainWindow -left-|> UiPart
 
 ResultDisplay --|> UiPart
+StatisticsDisplay --|> UiPart
+AppointmentListPanel --|> UiPart
+AppointmentCard --|> UiPart
 CommandBox --|> UiPart
 PersonListPanel --|> UiPart
 PersonCard --|> UiPart
```

Comment:
Hmm, perhaps all instances of "Person" should be changed to "Patient" for the diagrams.